packages/labs/src/components/select/select.tsx

```diff
@@ -46,6 +46,18 @@ export interface ISelectProps<T> extends IListItemsProps<T> {
     itemRenderer: (itemProps: ISelectItemRendererProps<T>) => JSX.Element;
 
     /**
+     * Current query value, for controlled usage.
+     * Providing this prop will put the component in controlled mode.
+     */
+    query?: string;
+
+    /**
+     * Callback invoked when the query value changes,
+     * through user input or when the filter is reset.
```

Comment:
We're woefully inconsistent at this, but can we align documentation to 80 chars? There's a VSCode plugin to help (`Align`).

Reply:
I'm fine with this. 80 chars would require two lines anyway and this is just so handsomely formatted 😍

Reply:
Unfortunately, prettier doesn't have an opinion on this matter. :)